opa/tests/test_opa.py

```diff
@@ -0,0 +1,102 @@
+from psycopg2.errors import CheckViolation, NotNullViolation, UniqueViolation
+
+from odoo import fields, Command
+from odoo.exceptions import ValidationError
+from odoo.tests import TransactionCase
+from odoo.tools import mute_logger
+
+
+class TestOpa(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        opa_manager_group = cls.env.ref('opa.group_manager')
```

Comment:
Use `browse_ref` to get the object or `ref` to get the id

I suggest:
```suggestion
        opa_manager_group_id = cls.ref('opa.group_manager')
```

Reply:
and change the references of `opa_manager_group.id ` to `opa_manager_group_id`